docs/migration/migrate_7_0.asciidoc

```diff
@@ -41,4 +41,10 @@ so we no longer need this workaround.
 *Impact:* Since we'll no longer send the `default_field` parameter in Kibana's query_string query, Elasticsearch
 will use the index setting instead. The default for the index setting is also `*`, so most users should not be impacted.
 If some of your indices have a non-default `default_field` setting, you may want to update it or re-add the parameter
-to Kibana's advanced setting.
\ No newline at end of file
+to Kibana's advanced setting.
+
+[float]
+=== Deprecated kibana.yml setting `xpack.monitoring.node_resolver` has been removed
+*Details:* This setting has been deprecated since 5.6, when it was explicitly recommended to use `uuid` as its value.
+
+*Impact:* The setting is no longer needed to use the best way to determine node uniqueness, its persistent UUID.
```

Comment:
It might be nice to clarify the impact with something like this (stolen from https://www.elastic.co/guide/en/kibana/6.3/xpack-monitoring.html):

> This setting is no longer necessary. If you enable {monitoring} across the Elastic Stack, a monitoring agent runs on each Elasticsearch node, Logstash node, Kibana instance, and Beat to collect and index metrics. Each node and instance is considered unique based on its persistent UUID, which is written to the path.data directory when the node or instance starts.